prog/kube-peers/main.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"k8s.io/client-go/1.4/kubernetes"
```

Comment:
I believe https://github.com/weaveworks/weave/pull/2551/files#diff-8903239df476d7401cf9e76af0252622R106 should do this.